media-libs/libplacebo/libplacebo-1.7.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://code.videolan.org/videolan/libplacebo.git"
+	inherit git-r3
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://code.videolan.org/videolan/libplacebo/-/archive/v${PV}/libplacebo-v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
PS: technically, that is unnecessary cosmetics (filename is vaild without modification)

Reply:
Oh, indeed. Changed